### PR TITLE
[feature] Dynamic configs

### PR DESCRIPTION
**Pull request by KostiantynPopovych**
This PR adds the ability to run the report using dynamic Babel and ESLint configs without the need to add required plugins to the `package.json`.

### CODE REVIEW

**Review by etagwerker — APPROVED** [review]
@KostiantynPopovych Looks good! Could you please resolve the conflicts in this PR?

File: lib/plato.js [hunk 1]
```diff
@@ -118,86 +131,89 @@ function inspect(files, outputDir, options, done) {
...
-			}
-		});
-		if (done) {
-			done();
-		}
-	}
-
-	if (!fileOutputDir) {
-		runReports(files, function() {
+  async function runReports(files, done) {
```
Comment (briandonahue) [inline lib/plato.js:142]: Minor suggestion, but breaking out some utility functions to break this long method up might be nice.

**Review by briandonahue — APPROVED** [review]
LGTM, made one minor comment

File: lib/plato.js (hunk 1 above)
Comment (KostiantynPopovych) [reply lib/plato.js:142]: @briandonahue, totally agree with you, but I think for now we are trying to come up with more required features, and then we can polish this and rebuild, it requires a lot.

**Pull request merged**